src/index.jsx

```diff
@@ -21,13 +27,15 @@ const crashReporter = () => next => action => {
 };
 
 const store = createStore(
-  combineReducers({ experiments }),
+  combineReducers({ experiments, date }),
```

Comment:
Oh I just realized, this one should be called 'dates' because it will be storing multiple dates.